cypress/support/commands.js

```diff
@@ -47,24 +47,39 @@ Cypress.Commands.add('login', (email, username, password) => {
   });
 });
 
-Cypress.Commands.add('createArticle', (title, description, body) => {
-  cy.getCookie('auth').then((token) => {
-    const authToken = token.value;
+Cypress.Commands.add('createArticle', (user, article) => {
+  cy.request({
+    method: 'POST',
+    url: '/api/users/login',
```

Comment:
It seems like the login request is being made again in the 'createArticle' command. If the user is already logged in, this step might be redundant. Consider reusing the authentication token from the 'login' command if possible.